app/client-app/src/app/pages/Wordlist/components/WordlistSortingAndFiltering.jsx

```diff
@@ -0,0 +1,41 @@
+import { Flex, VStack, Input, Text, HStack, Box } from "@chakra-ui/react";
+import SortPanel from "components/shared/SortPanel";
+
+const WordlistSortingAndFilteringPanel = ({
+  data,
+  sorting,
+  onSort,
+  filter,
+  setFilter,
+}) => {
+  return (
+    <Flex alignItems="center" flexDirection="column">
+      {Object.keys(data).length && (
```

Comment:
```suggestion
      {Object.keys(data).length > 0 && (
```
Conditional rendering on .length can end up rendering "0" if data is empty, as it's still truthy.